stable/vpa/.gitignore

```diff
@@ -0,0 +1,2 @@
+charts/
+Chart.lock
```

Comment:
I don't think it's best practice to exclude this, is it?

Reply:
I oriented myself on the other examples in this repository.
I will change that, thanks for the comment.